### ISSUE TITLE
Inconsistency of ResNet Input Format in Docs

### ISSUE THREAD

**Issue by bbartoldson** [tensorflow/hub#92]
According to [the Hub docs](https://www.tensorflow.org/hub/modules/google/imagenet/resnet_v2_152/feature_vector/1):

> This TF-Hub module uses the TF-Slim implementation of resnet_v2_152 with 152 layers...  The input images are expected to have color values in the range [0,1], following the common image input conventions.

According to [the Slim docs](https://github.com/tensorflow/models/tree/master/research/slim):

> ResNet V2 models use Inception pre-processing

[Lines 280-281 of Inception pre-processing normalize the data to values in [-1,1]](https://github.com/tensorflow/models/blob/master/research/slim/preprocessing/inception_preprocessing.py), not [0,1] as the Hub docs say. 

In case it's helpful for determining the correct pre-processing: When I use [-1,1] normalization, the scale of the ResNet features is larger (maximum of 113, mean of 3). Using [0,1] normalization, the scale of the ResNet features is smaller (maximum of 14, mean 0.7).

**Comment by arnoegw** [comment]
TL;DR: Both docs are correct, but only the Hub doc matters for the Hub module.

Details:

In the Slim library, the input range depends on the network architecture, and sometimes even on the particular checkpoint. The Slim docs you cite talk about that.

In TensorFlow Hub, the input range of all modules has been fixed to [0,1], which is what you get right away from `tf.image.decode_image(...)` followed by `tf.image.convert_image_dtype(..., tf.float32)`. (Using float inputs and not uint8 helps in case you'd want to backprop through a module.) Any further scaling is an implementation detail hidden inside the module. (For example, the Hub modules made from Slim nets with Inception-style preprocessing internally convert from [0,1] to [-1,+1] before running the actual Slim net.) That way, users of Hub modules are freed from figuring out which input range to use with which architecture or pretrained checkpoint.
